app/models/status.rb

```diff
@@ -250,6 +255,8 @@ def decrement_count!(key)
   before_validation :set_conversation
   before_validation :set_local
 
+  before_save :set_poll_id
```

Comment:
We can use a normal association instead of this, now, can't we?

Reply:
We have this throwaway inverse reference to polls so that during eager-loading we can skip queries on the polls table when no statuses have polls, which I believe would be the most common case.